scripts/core/adapter.yml

```
@@ -9,177 +9,9 @@
 #
 --- #--------------------------------------------------------------------------
 type: header
-desc: "Intel $OneApi Unified Runtime APIs for Runtime"
+desc: "Intel $OneApi Unified Runtime APIs for Adapter"
 ordinal: "0"
```

currently adapter comes before loader in the rendered contents menu, changing the ordinal should fix that.

```suggestion
ordinal: "1"
```